Automattic/wp-calypso
# Add AI to Complete feature copy

## Proposed Changes

* Add AI to Complete feature copy

## Why are these changes being made?

We are adding Jetpack AI to the Complete bundle: pbNhbs-cbd-p2

## Testing Instructions

1. Open the Calypso Blue live link and go to `/checkout/jetpack/jetpack_complete`
2. Ensure you see `AI Assistant` as a feature to complete
![Screenshot 2024-12-11 at 12 52 07 PM](https://github.com/user-attachments/assets/bc3cbcf6-8838-4ced-ba38-66d5170a2cc7)
3. Open the Calypso Green live link and go to `/pricing#jetpack_complete`
4. Ensure you see AI as a feature
![Screenshot 2024-12-11 at 12 53 28 PM](https://github.com/user-attachments/assets/a9c17705-0b8e-46d6-9a11-b4d6ea162a5a)
5. Now go to `/features/comparison`
6. Ensure you see `High request capacity` for AI under the Complete plan
![Screenshot 2024-12-11 at 12 54 35 PM](https://github.com/user-attachments/assets/278bdf1a-852c-4ff2-af04-94b1f7ebd41d)


## Pre-merge Checklist
